### ISSUE TITLE
t6 raid bosses / spell interrupt

### ISSUE THREAD

**Issue by kauski** [FeenixServerProject/Archangel_2.4.3_Bugtracker#3826]
Teron gorefiend: does not show targetoftarget when he casts  Shadow of Death, as it can be interrupted by the targeted player, mages with invisibility? hunters feign death 100% and rogues vanish.

Also shariva posted earlier about Anetheron infernal cast being interrupted by hunters mages rogues, it worked normally before, but you believed a guy like shariva who doesnt know mechanics of tier6 raid bosses, you changed anetheron that its no longer possible to interrupt his infernal cast.

So basically you are able to interrupt anetheron infernal spawn cast with some classes, hunters, mages and rogues on the live wow back on TBC version.

PLease dont believe everything that shariva says hes a mage and he thinks he is right but hes not.

**Comment by kauski** [comment]
-^ infernal/shadow of death are similiar interruptable such as archimonde airburst by some classes.

**Comment by Applescruffs** [comment]
Maybe you should leave a comment on the specific issue that you think they fixed incorrectly with some proof?
Instead of just expecting the developers to believe that you, instead of Shariva, are correct.
